tola/translation_data/README-TRANSLATION.md

```diff
@@ -0,0 +1,200 @@
+```
+
+The `msgcat` command often results in an output file with an invalid format. This is usually because the two headers are different and `msgcat` isn't able to pick one.  The easiest way to check for errors is to open the file in POEdit, which will generate an error if the .po file format is invalid. If it is invalid you will need to edit the .po file in a text editor and pick one of the two headers.  See the note below on `msgcat` for additional details.
+
+Once you have resolved this potential issue, `django_fr_to_translators.po` and `djangojs_fr_to_translators.po` will be ready to send to the translators.
+
+
+## Process raw translated files
+When the files come back from the translators, transfer them into the `<temp>` directory.
+```bash
+toladata$ cp ~/Downloads/django_fr_to_translators--done.po <tempdir>
+toladata$ cp ~/Downloads/djangojs_fr_to_translators--done.po <tempdir>
+```
+
+You will need to do some quality checks on them before combining them with the translated strings:
+- Check for alerts in poedit.  The translators will occasionally miss things like spaces at the beginning or end of strings.  POEdit is good at catching those and warning you about them.
+- Sometimes the translators will forget to remove the fuzzy mark on strings.  In POEdit, you can use the "Needs Work" slider near the translated string to turn this off, provided that the string has actually been translated of course.
+- Check any plural strings to be sure the plural value was translated along with the singular.
+- Check any strings with code marks to ensure they survived the translation process and were appropriately handled by the translators.
+
+Once you're satisfied with the new translations, they should be merged into the codebase. There are two ways to do this.  If it has been a long time since you sent the files to the translators and/or if there have been updates to the dev branch .po files that you wish to preserve, Option 1 is probably better, since it uses the dev branch .po files as a base.  If there haven't been any changes in the dev branch .po files that you need to keep and/or if there have been a lot of updates that you want to dump, Option 2 may be better because it regenerates any missing translations from scratch.
+
+### Option 1: Use the dev branch .po files
+You can use `msgmerge` to update the dev branch .po files with the new translations received from the translators.  First merge the files together, then copy the merged files from `<tempdir>` to the `<locale>` folder.
+```bash
+$ msgmerge -o <tempdir>/django_fr_merged.po <tempdir>/django_fr_to_translators--done.po <locale>/fr/LC_MESSAGES/django.po
+$ msgmerge -o <tempdir>/djangojs_fr_merged.po <tempdir>/djangojs_fr_to_translators--done.po <locale>/fr/LC_MESSAGES/djangojs.po
+$ cp <tempdir>/django_fr_merged.po <locale>/fr/LC_MESSAGES/django.po
+$ cp <tempdir>/djangojs_fr_merged.po <locale>/fr/LC_MESSAGES/djangojs.po
+```
+
+Your translations should not be all done!
```

Comment:
Should not be or should now be all done?

Reply:
Lol, _should_.

Reply:
Oh, and actually, just deleted the whole line, since it's premature.